boards.txt

```diff
@@ -544,6 +544,18 @@ Disco.build.core=arduino
 Disco.build.board=Disco
 Disco.build.extra_flags=-D{build.product_line} {build.enable_usb} {build.xSerial}
 
+# DISCO_B-G474E-DPOW1 board
+Disco.menu.pnum.DISCO_B-G474E-DPOW1=DISCO_G474E_DPOW1
+Disco.menu.pnum.DISCO_B-G474E-DPOW1.node="No_mass_storage_for_this_board_Use_STLink_upload_method"
```

Comment:
.node is the name of the computer mounted disk when you plug STlink USB cable to your computer.
I think this board support Mass Storage. But I don't have a board with me to verify, Chan you check this ?